pubsub/observer.go

```diff
@@ -0,0 +1,26 @@
+package pubsub
```

Comment:
@nivBlox why do we need it as a dedicated package in the root dir? I think this should be part of the eth1 package, not a separate one @lior-blox

Reply:
eth1 only implement that. 
In my opinion we need to move this pkg under utils